playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -317,6 +317,16 @@
     - install
     - install:app-requirements
 
+- name: Install private node dependencies
+  shell: "easy_install --version && npm clean-install"
```

Comment:
I know this isn't the right command. I'm not sure how to

- how to separate the installation of public and private npm modules, because the current command to install npm modules is npm clean-install.
- how a package-lock.json is generated, since npm clean-install will need to compare against it